src/button-card.ts

```diff
@@ -140,23 +52,7 @@ class ButtonCard extends LitElement {
   }
 
   protected shouldUpdate(changedProps: PropertyValues): boolean {
-    return hasConfigOrEntityChanged(this, changedProps);
-  }
-
-  private getFontColorBasedOnBackgroundColor(backgroundColor: string): string {
-    let localColor = backgroundColor;
-    if (backgroundColor.substring(0, 3) === 'var') {
-      localColor = window.getComputedStyle(document.documentElement)
-        .getPropertyValue(backgroundColor.substring(4).slice(0, -1)).trim();
-    }
-    const colorObj = new TinyColor(localColor);
-    let fontColor = ''; // don't override by default
-    if (colorObj.isValid && colorObj.getLuminance() > 0.5) {
-      fontColor = 'rgb(62, 62, 62)'; // bright colors - black font
-    } else {
-      fontColor = 'rgb(234, 234, 234)';// dark colors - white font
-    }
-    return fontColor;
+    return helpers.hasConfigOrEntityChanged(this, changedProps);
```

Comment:
I'd prefer you remove `helpers.` and instead import the function explicitly. Just my preference though; your call